src/common/constants.ts

```diff
@@ -0,0 +1,13 @@
+export enum OPERATIONS {
+  CREATE = 'CREATE',
+  READ = 'READ',
+  UPDATE = 'UPDATE',
+  DELETE = 'DELETE',
+}
+
+export const HOST: string = 'localhost';
+export const PORT: number = 3000;
+export const DB_PORT: number = 3307;
+export const DB_USERNAME: string = 'postgres';
+export const DB_PASSWORD: string = 'root1234';
+export const DB_DATABASE: string = 'userdb';
```

Comment:
Convert this file to the NestJS config
https://docs.nestjs.com/techniques/configuration#using-the-configservice